keystone/server.sls

```diff
@@ -204,6 +204,7 @@ keystone_syncdb:
 
 {% if server.tokens.engine == 'fernet' %}
 
+{%- if not grains.get('noservices', False) %}
```

Comment:
tend to agree, obviously not needed, as the other PR now is capable to run full test scope without it. Will close soon.